package.json

```diff
@@ -20,5 +20,23 @@
     "homepage": "https://github.com/nick-w-nick/eslint-config-lintification#readme",
     "keywords": ["eslint", "lintification", "lint", "typescript", "nodejs", "backend"],
     "author": "https://github.com/nick-w-nick",
-    "license": "MIT"
+    "license": "MIT",
+    "dependencies": {
+        "eslint": "^8.56.0",
+        "@stylistic/eslint-plugin": "2.10.1",
+        "@typescript-eslint/eslint-plugin": "6.7.4",
+        "@typescript-eslint/parser": "6.7.4",
+        "eslint-config-airbnb": "19.0.4",
+        "eslint-config-airbnb-base": "15.0.0",
+        "eslint-config-airbnb-typescript": "17.1.0",
+        "eslint-import-resolver-typescript": "3.6.1",
+        "eslint-plugin-destructuring-newline": "0.0.2",
+        "eslint-plugin-etc": "2.0.3",
+        "eslint-plugin-file-progress": "1.3.0",
+        "eslint-plugin-import": "2.28.1",
+        "eslint-plugin-import-newlines": "1.3.4",
+        "eslint-plugin-jest": "27.4.2",
+        "eslint-plugin-simple-import-sort": "10.0.0",
+        "typescript-eslint": "8.7.0"
+    }
```

Comment:
I'm uncertain if these need to be dependencies, peer dependencies, or dev dependencies (or multiple). Will triaging that locally once the rules are fleshed out more to confirm.

Reply:
Plugins need to be peer dependencies according to eslint documentation. Additionally, I found that `eslint-import-resolver-typescript` needed to be a peer dependency as well, otherwise errors like this would get spammed to the console in every TypeScript file:
```
1:1   error  Resolve error: typescript with invalid interface loaded as resolver      import/no-relative-packages
  1:27  error  Unexpected use of file extension "js" for "./path-to-your-file.js"  import/extensions
```